applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -139,6 +174,13 @@ static void advertising_process(struct k_work *work)
 		adv_param.id = BT_ID_DEFAULT;
 		adv_param.options |= BT_LE_ADV_OPT_DIR_ADDR_RPA;
 
+		/* Clear ADV data set before update to direct advertising */
+		ret = bt_le_ext_adv_set_data(adv_ext, NULL, 0, NULL, 0);
+		if (ret) {
```

Comment:
I think the point in Alexanders previous comment was that we should actually not do if (ret), but in stead do if (ret < 0).
We do if (ret) throughout the whole nrf5340_audio application though..

Reply:
Yes, but we decided to have another task for clean-up all the code for making it consistent